Sources/Plasma/FeatureLib/pfPython/pyVaultNode.cpp

```diff
@@ -580,13 +580,13 @@ PyObject * pyVaultNode::GetNode2( uint32_t nodeID ) const
     PYTHON_RETURN_NONE;
 }
 
-PyObject* pyVaultNode::FindNode( pyVaultNode * templateNode )
+PyObject* pyVaultNode::FindNode( pyVaultNode * templateNode, unsigned int maxDepth )
```

Comment:
Is there a reason this doesn't also use `unsigned int maxDepth = 1` like the header file does? I'm not sure what the rules or convention around optional parameters in C++ are, but in C# I think I would normally write it in both places, e.g., in a method declaration in an interface and in the method definition in the class implementing the interface.

Reply:
In C and C++, the default argument should only be listed in the declaration (the header file with the class definition in this case). Including it in the method definition is actually an error on some compilers (eg gcc).